src/components/Field/Field.test.js

```diff
@@ -83,39 +83,6 @@ describe('field rendering', () => {
   })
 })
 
-describe('fertilize-mode class', () => {
-  test('is not present when fieldMode != FERTILIZE', () => {
-    expect(component.find('.Field').hasClass('fertilize-mode')).toBeFalsy()
-  })
-
-  test('is present when fieldMode == FERTILIZE', () => {
-    component.setProps({ fieldMode: fieldMode.FERTILIZE })
-    expect(component.find('.Field').hasClass('fertilize-mode')).toBeTruthy()
-  })
-})
-
-describe('plant-mode class', () => {
-  test('is not present when fieldMode != PLANT', () => {
-    expect(component.find('.Field').hasClass('plant-mode')).toBeFalsy()
-  })
-
-  test('is present when fieldMode == PLANT', () => {
-    component.setProps({ fieldMode: fieldMode.PLANT })
-    expect(component.find('.Field').hasClass('plant-mode')).toBeTruthy()
-  })
-})
-
-describe('harvest-mode class', () => {
-  test('is not present when fieldMode != HARVEST', () => {
-    expect(component.find('.Field').hasClass('harvest-mode')).toBeFalsy()
-  })
-
-  test('is present when fieldMode == HARVEST', () => {
-    component.setProps({ fieldMode: fieldMode.HARVEST })
-    expect(component.find('.Field').hasClass('harvest-mode')).toBeTruthy()
-  })
-})
```

Comment:
These and the removed tests below are obviated by `src/shell/field.test.js`, introduced by this PR: https://github.com/jeremyckahn/farmhand/blob/2b8721e394c9e3eae24263f42ec29ff6f8d5d110/src/shell/field.test.js#L15-L81